gdcdictionary/schemas/pathology_detail.yaml

```diff
@@ -351,6 +381,8 @@ properties:
       - Supraclavicular
       - Unknown
       - Not Reported
+      - Aortic
+      - Pelvis, NOS
```

Comment:
it says "Pelvic" in the jira.

Reply:
Thanks, thought I had updated JIRA; when conferring with Megan Stine @ IMS, it was agreed enum should be "Pelvis, NOS".